docs/ServiceConfiguration.md

```diff
@@ -0,0 +1,45 @@
+# Service configuration
+
+Vehicle recall service is able to use few environment variable. They are used to set properly application logger.
+
+```
+SERVICE_NAME
+```
+This parameter must be set during deployment. It contains service name "Vehicle recalls api" and it is added to all logs generated by service. If not set then "NotSet" string is generated.
+
+```
+SERVICE_ENV
+```
+This parameter must be set during deployment. It contains environment name and it is added to all logs generated by service. If not set then "NotSet" string is generated.
```

Comment:
`must` implies that the service would not work without it. `should` would be more suitbale if a default is set

Reply:
good point